### PR TITLE
Feature D23UPE6-18 : Back-End: API to PUT/EDIT a Warehouse

### PR DESCRIPTION
**Pull request by charuzain**
- Implemented the route to edit a warehouse
- Include necessary validations

### CODE REVIEW

File: index.js
Comment (hilarykhc) [inline index.js:1]: why the name change for routes?

**Review by hilarykhc — COMMENTED** [review]
code looks good and well structured. 
Why the name change in routes?

File: index.js
Comment (charuzain) [reply index.js:1]: I was reviewing backend tickets, I noticed that the endpoints specified in the requirements differed from the names we were using in our routes. Therefore, I updated our routes to align with the specified endpoints in the requirements.

Comment (hilarykhc) [reply index.js:1]: Thank you for updating the routes!

**Pull request merged**